common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptDao.java

```diff
@@ -8,6 +8,10 @@
 
 public interface ConceptDao extends CrudRepository<Concept, Long> {
 
+    @Query(nativeQuery=true, value="select c.* from concept c join concept_relationship cr on c.concept_id=cr.concept_id_2 " +
+            "where cr.concept_id_1=?1 and cr.relationship_id='Maps to' ")
+    Concept findStandardConcept(long concept_id);
```

Comment:
Is this definitely going to be limited to standard concept relationships? Is there guaranteed to only be one of these?

Reply:
I got to check those.